internal/resources/resource_lb_virtual_servers.go

```diff
@@ -74,14 +70,15 @@ func LoadBalancerVirtualServers() *schema.Resource {
 					"COOKIE",
 				}, false),
 				Optional:    true,
-				Description: "persistence type for Network loadbalancer virtual server",
+				Description: "Persistence type for Network loadbalancer virtual server",
 			},
 			"cookie_persistence_profile":   schemas.CookiePersProfileSchema(),
 			"sourceip_persistence_profile": schemas.SourceipPersProfileSchema(),
 			"ssl_server_cert": {
-				Type:        schema.TypeInt,
-				Required:    true,
-				Description: "ID of the ssl_server_cert. Use " + DSLBVirtualServerSslCert + "datasource to obtain the id  here",
+				Type:     schema.TypeInt,
+				Required: true,
+				Description: "SSLServerCert is needed only for https based load balancer" +
+					"ID of the ssl_server_cert. Use " + DSLBVirtualServerSslCert + "datasource to obtain the id  here",
```

Comment:
Can you make these statements more use readable ? the grammar is totally messed up here. applies to all the places. I would suggest u to read it before reaching out for review. This comment applies to all the places.

Reply:
updated